cneuromax/projects/friends_language_encoder/litmodule_peft.py

```diff
@@ -0,0 +1,89 @@
+""":class:`FriendsFinetuningModel`."""
+
+from dataclasses import dataclass
+from typing import Any
+
+from jaxtyping import Num
+from peft.config import PeftConfig
+from peft.mapping import get_peft_model
+from torch import Tensor
+from transformers.tokenization_utils_base import BatchEncoding
+
+from cneuromax.fitting.deeplearning.litmodule import (
+    BaseLitModule,
+    BaseLitModuleConfig,
+)
+from cneuromax.projects.friends_language_encoder.peftmodule import (
+    PEFTLitModule,
+)
+from cneuromax.utils.beartype import one_of
+
+
+@dataclass
+class FriendsLitModuleConfig(BaseLitModuleConfig):
+    """Holds :class:`FriendsLitModule` config values.
+
+    Args:
+        layer_name: layer to unfreeze
+    """
+
+    layer_names: str = "${layer_names}"
```

Comment:
1. are you sure this is a string? looks like it's a list from your docstring
2. donc set the config to `${layer_names}` if you need a default set it to `field(default_factory=list)` if it's a list of string, or `""` if it's a string

Reply:
alright then smth like `layer_names: list[str] = field(default_factory=list)` should do (means the default `[]` empty list). 
See https://docs.python.org/3/library/dataclasses.html#dataclasses.field